Restructure documentation in test-infra (#110)
kyma-project/test-infra: prow/scripts/cluster-integration/README.md

# Cluster Integration

## Overview

> The folder contains scripts that are responsible for performing a cleanup of Google Cloud Platform (GCP) objects built as part of the integration tests that run on pull requests.

Reviewer: Scripts in that folder are not only responsible for cleanup. For example `pr-gke-integration.sh` create cluster.

# Cluster Integration

## Overview

The folder contains scripts that are involved in the preparation and removal of the cluster setup on Google Kubernetes Engine (GKE).

### Project structure

<!-- Update the folder structure each time you modify it. -->

The structure of the folder looks as follows:

```

>   ├── create-dns-entry.sh               # The script that creates a DNS entry on Google Cloud Platform (GCP) that points to the GKE cluster.

Reviewer 2: Also: dots

>   ├── delete-dns-record.sh              # The script responsible for removing a DNS record from GCP after completion of integration tests.

Reviewer 2: ```suggestion
  ├── delete-dns-record.sh              # The script which removes a DNS record from GCP after completion of integration tests.
```
